commands/synonym_command.rb

```diff
@@ -0,0 +1,41 @@
+module Commands
+  class SynonymCommand
+    class << self
+      include Discordrb::Webhooks
+
+      def name
+        :synonym
+      end
+
+      def attributes
+        {
+          min_args: 1,
+          usage: 'synonym term',
+          description: 'Return a list of synonyms for a term.'
+        }
+      end
+
+      def command(event, *term)
+        return if event.from_bot?
+
+        request = 'https://api.datamuse.com/words?rel_syn=' << term[0]
+        response = RestClient.get(request)
+        body = JSON.parse response
+
+        wordset = body.sort { |word| word['score'] }
```

Comment:
This variable isn't used.

Reply:
whoops, its supposed to have the words sorted by score, so it should be something along the lines of body.sort_by! { |word| word['score'] }, my mistake.